homeassistant/components/media_player/philips_js.py

```diff
@@ -165,6 +167,13 @@ def mute_volume(self, mute):
         if not self._tv.on:
             self._state = STATE_OFF
 
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        if self._volume != volume:
+            self._tv.setVolume(volume)
+            if not self._tv.on:
+                self._state = STATE_OFF
```

Comment:
We should not do this. Home Assistant will query the TV for an update and if the TV did turn on, it will be fetched then.